utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -799,15 +800,25 @@ Describe 'Module tests' -Tag 'Module' {
                 }
 
                 if ($incorrectParameters.Count -gt 0) {
-                    $warningMessage = 'All parameters which are of type [object] or [array-of-objects] should implement a user-defined, or resource-derived type. Found incorrect items '
-                    Write-Warning ("$warningMessage`n- {0}`n" -f ($incorrectParameters -join "`n- "))
 
-                    Write-Output @{
-                        Warning = ("$warningMessage<br>- <code>{0}</code><br>" -f ($incorrectParameters -join '</code><br>- <code>'))
+                    $versionFilePath = Join-Path (Split-Path $templateFilePath) 'version.json'
+                    $hasVersionFile = Test-Path $versionFilePath
```

Comment:
Wondering if we can leverage the versionFileExists at row [540](https://github.com/Azure/bicep-registry-modules/pull/5394/files#diff-9919fd54ac0072d19fc1983df20b7363f589a83758d0ae2db89ec5b6294c8ff5R540)

Reply:
Fair, just double checking if conscious choice 🙂 Please revolve

Reply:
Updated